theme/layout/breakpoints.less

```diff
@@ -1,2 +1,20 @@
 // Breakpoint
 @responsive-breakpoint: 740px;
+
+.smallDeviceOnly(@rules) {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    @rules();
+  }
+}
+
+.aboveSmallDevice(@rules) {
+  @media only screen and (min-width: @responsive-breakpoint) {
+    @rules();
+  }
+}
+
+.classicOnly(@rules) {
```

Comment:
I realised that this mixin is only used in `Header.less`, so I'm making it local to that file, and renaming it to `classicDesktopOnly`.